mnefun/_mnefun.py

```diff
@@ -822,38 +863,141 @@ def run_sss_positions(fname_in, fname_out, host='kasga', opts='', port=22):
         raise IOError('input file not found: %s' % fname_in)
     if not op.isdir(op.dirname(op.abspath(fname_out))):
         raise IOError('output directory for output file does not exist')
-    fnames_in = [fname_in]
-    for ii in range(1, 11):
-        next_name = op.splitext(fname_in)[0] + '-%s' % ii + '.fif'
-        if op.isfile(next_name):
-            fnames_in.append(next_name)
-        else:
-            break
```

Comment:
This func was quite buggy as it was, e.g. this particular loop was unclear to and I suspect the source of the most recent error @jyeatman reported concerning missing raw files with modified basenames as per `next_name`. WDYM by auto detection? The `dict` implementation should account for split files > raw-1.fif for a given run_index. In the past, e.g., in `get_raw_names` we resorted to reg expression to find split files. This loop wasn't really working, and by the time I figured out what you were trying to do, I had the `dict` approach working.